src/goto-instrument/accelerate/disjunctive_polynomial_acceleration.cpp

```diff
@@ -399,13 +403,14 @@ bool disjunctive_polynomial_accelerationt::find_path(patht &path)
 bool disjunctive_polynomial_accelerationt::fit_polynomial(
   exprt &var,
   polynomialt &polynomial,
-  patht &path)
+  patht &path,
+  guard_managert &guard_manager)
```

Comment:
It may be better to make this a class member somewhere rather than passing it around all across the acceleration code?

Reply:
For readability or performance?